[2025-02-24T05:05:32.026+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T05:05:32.056+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_1.get_name scheduled__2025-02-22T00:00:00+00:00 [queued]>
[2025-02-24T05:05:32.072+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_1.get_name scheduled__2025-02-22T00:00:00+00:00 [queued]>
[2025-02-24T05:05:32.073+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T05:05:32.094+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): get_name> on 2025-02-22 00:00:00+00:00
[2025-02-24T05:05:32.102+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2306) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T05:05:32.104+0000] {standard_task_runner.py:72} INFO - Started process 2319 to run task
[2025-02-24T05:05:32.102+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_1', 'get_name', 'scheduled__2025-02-22T00:00:00+00:00', '--job-id', '251', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmp_4bo9crp']
[2025-02-24T05:05:32.106+0000] {standard_task_runner.py:105} INFO - Job 251: Subtask get_name
[2025-02-24T05:05:32.170+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_1.get_name scheduled__2025-02-22T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T05:05:32.280+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag_1' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-22T00:00:00+00:00'
[2025-02-24T05:05:32.282+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T05:05:32.329+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-24T05:05:32.341+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T05:05:32.342+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag_1, task_id=get_name, run_id=scheduled__2025-02-22T00:00:00+00:00, execution_date=20250222T000000, start_date=20250224T050532, end_date=20250224T050532
[2025-02-24T05:05:32.402+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T05:05:32.427+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T05:05:32.429+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
